verify.go

```diff
@@ -158,7 +152,7 @@ func (c Config) validColumnTarget(columnName string) bool {
 	return false
 }
 
-func (c Config) runTestQueriesOnTarget(ctx context.Context, logger *logrus.Entry, conn *pgx.Conn, schemaTableHashes SingleResult) (SingleResult, error) {
```

Comment:
I'm guessing because the linter flagged that we never return a non-nil error here?

Reply:
correct